benchmark/rclcpp/topic/publisher-stress-test.cpp

```diff
@@ -68,9 +68,9 @@ int main(int argc, char* argv[]) {
       *height_dim, *width_dim, *channel_dim};
   layout->data_offset = 0;
 
-  auto msg = std::make_shared<std_msgs::msg::UInt8MultiArray>();
-  msg->layout = *layout;
-  msg->data = std::vector<uint8_t>(1024 * amount, 255);
+  auto msg = std_msgs::msg::UInt8MultiArray();
+  msg.layout = *layout;
+  msg.data = std::vector<uint8_t>(1024 * amount, 255);
```

Comment:
This change worked around a compilation error that I was stuck trying to resolve directly. So I went this direction. Please advise if there is an alternative change to the original code that will resolve the compilation problem.

Reply:
it looks good.